content/300-accelerate/650-troubleshoot.mdx

```diff
@@ -110,9 +110,11 @@ This error indicates that Prisma Accelerate cannot establish a connection to you
 
 #### Database Not Publicly accessible
 
-Prisma Accelerate currently requires the database to be publicly accessible. If your database is behind a VPC, or access is restricted to specific IP addresses, this error may occur.
+If your database is behind a VPC, or access is restricted to specific IP addresses, this error may occur.
 
-**Suggested solution:** Ensure your database is publicly accessible, if not, allow public access. Please note this is only an interim requirement, Static IPs are coming soon and will alleviate the need for complete public access if that’s not desirable.
+**Suggested solution:** You'll have to enable static IP within your Accelerate configuration in your project environment. After you opt-in to use Accelerate with static IP enabled, add the IP address you receive in the database firewall IP allowlist or permit it access to your private network. This will ensure that Accelerate can securely access your database.
```

Comment:
```suggestion
**Suggested solution:** Enable static IP within your Accelerate configuration in your project environment. After you enable static IP, add the IP address you receive in the database firewall IP allowlist or permit it access to your private network. This will ensure that Accelerate can connect to your database.
```